cleanlab_studio/cli/cleanset/download.py

```diff
@@ -62,15 +63,10 @@ def download(
     CleanlabSettings.init_cleanlab_dir()
     api_key = config.get_api_key()
     progress("Downloading Cleanlab columns...")
-    rows = api_service.download_cleanlab_columns(api_key, cleanset_id=id, all=all)
-
-    if all:
```

Comment:
It doesn't seem like this "all" argument is actually of use anymore -- consider getting rid of it?

Reply:
Seems to have been removed due to the refactor to have names on backend side